docker/install/ubuntu_install_qemu.sh

```diff
@@ -20,4 +20,473 @@ set -e
 set -u
 set -o pipefail
 
-apt-get -y install qemu-system
+sudo sed -i '/deb-src/s/^# //' /etc/apt/sources.list
+apt update
+apt-get -y build-dep qemu
+
+cat <<EOF | gpg --import
+-----BEGIN PGP PUBLIC KEY BLOCK-----
```

Comment:
Is it possible to avoid adding the pubkey here and download the key from a place?

Reply:
changed to `--recv-keys`